### ISSUE TITLE
Symfony 3.3 error  

### ISSUE THREAD

**Issue by diegograssato** [rdohms/dms-filter-bundle#30]
On upgrade to symfony 3.3 error was ocurred.

```
PHP Fatal error:  Uncaught Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: Invalid key "filter" found in arguments of method "__construct()" for service "dms.filter.type_extension": only integer or $named arguments are allowed. in /Dados/open_solutions/web/api-ciand-example/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/ResolveNamedArgumentsPass.php:47
```

**Comment by rdohms** [comment]
@diegograssato can you tests a branch for me to see if it works?

change your composer to include `"dev-no-named-arguments@dev" and let me know if that fixes it, thanks!

**Comment by diegograssato** [comment]
Hi @rdohms,
It worked, I saw that you opened a PR for the branch master, I hope you will be merged soon, thank you for your work.

This PR solve: https://github.com/rdohms/dms-filter-bundle/pull/31

**Comment by rdohms** [comment]
@diegograssato just released v3.2 with this fix, hope that solves it.
